src/handlers/comment/handlers/allow.ts

```diff
@@ -1,62 +1,63 @@
-import { upsertAccessControl } from "../../../adapters/supabase";
-import { getBotContext, getLogger } from "../../../bindings";
-import { getUserPermission } from "../../../helpers";
+import { getAdapters, getBotContext, getLogger } from "../../../bindings";
+import { isUserAdminOrBillingManager } from "../../../helpers";
 import { Payload } from "../../../types";
 
-export const setAccess = async (body: string) => {
+export async function setAccess(body: string): Promise<string> {
```

Comment:
Traditional function declarations are superior for general use. The only time arrow functions should be used are for tiny functions that are inlined.